integrationExamples/gpt/load_pbjs_dfp_concurrently.html

```diff
@@ -5,47 +5,38 @@
 * This is the optimized way of loading prebid.js. It loads both DFP and Prebid concurrently.
 * Publishers may wish to tweak the the MAX_RETRIES  and polling interval ms as desired to mitigate issue https://github.com/prebid/Prebid.js/issues/419
 */
-
+    
 //timeout to control how long bidders have to respond.
 var PREBID_TIMEOUT = 700;
-//how many times to attempt to load DFP if services have not been enabled
-var MAX_RETRIES = 20;
 
 var pbjs = pbjs || {};
 pbjs.que = pbjs.que || [];
-pbjs.retries = 0;
 var googletag = googletag || {};
 googletag.cmd = googletag.cmd || [];
 googletag.cmd.push(function () {
     googletag.pubads().disableInitialLoad();
 });
+
+// Load the Prebid Javascript Library Async. We recommend loading ASAP in page
+loadScript('//acdn.adnxs.com/prebid/not-for-prod/prebid.js');
+//load DFP async.
+loadScript('//www.googletagservices.com/tag/js/gpt.js');
```

Comment:
you can actually just use the standard `script` tag with the async attribute now since we are no longer supporting < IE 10. 
Ex: `<script async src = "//www.googletagservices.com/tag/js/gpt.js"></script>`

This will be more performant.